.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Awesome job! I'll be making a series of suggestion to make the template look better when the markdown is rendered. Keep it mind that you can take them as you see fit